Package.resolved

```diff
@@ -64,6 +64,15 @@
           "version": null
         }
       },
+      {
+        "package": "libcmark_gfm",
+        "repositoryURL": "https://github.com/KristopherGBaker/libcmark_gfm",
```

Comment:
Note to self: see how this points to the official cmark-GFM repo, i.e. how are we getting updates.

Reply:
So this directly depends on github/cmark-gfm by copying sources into the repo. cmark-gfm is on 0.29.0 and so is the Swift adapter. The latest release of cmark-gfm is from 8 Apr 2019 and there've been only two upstream releases in 2019, so it's not been changing a lot recently.

Should be pretty straightforward to update this dependency ourselves even if the maintainer doesn't, and it doesn't seem likely that there are going to be many changes to begin with.